src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img class="logo" src="images/logo.png"
+      alt="MOYO">
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="links is-active" href="1">Apple</a>
```

Comment:
Fix classes and href link everywhere
```suggestion
            <a class="nav__link is-active" href="#apple">Apple</a>
```